test/extended/testdata/deployments/custom-deployment.yaml

```diff
@@ -26,6 +26,7 @@ spec:
         echo Halfway
         openshift-deploy
         echo Finished
+        sleep 1
```

Comment:
Heh, I didn't realize you put that in. I wondered how this test was passing without https://github.com/docker/docker/pull/29863 (which isn't fullproof, but seemed to work every time I tried it using a docker with this patch).